ex04.ex

```diff
@@ -55,7 +55,9 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min(list) do
```

Comment:
## -1

Again with the`if`  :)

~~~ elixir
  def min(list), do: reduce(list, &min_of/2)
  def min_of(a, b) when a < b, do: a
  def min_of(a, b), do: b
~~~

Why is this better? Because it abstracts the processing of the list from what it done to each element. This means that changing it becomes clearer.